Tests/SonarRunner.Shim.Tests/SonarRunnerWrapperTests.cs

```diff
@@ -171,20 +167,85 @@ public void SonarRunner_SensitiveArgsPassedOnCommandLine()
             Assert.IsTrue(userPwdIndex < standardArgsIndex && userPwdIndex < propertiesFileIndex, "User arguments should appear first");
         }
 
+        [TestMethod]
+        public void WrapperError_Success_NoStdErr()
```

Comment:
I could compact all 6 tests into one, but I'd have to do a more complex setup - i.e. clean the sonar.runner.bat or use sub-dirs. Doesn't seem worth it.